crates/sqlez/src/connection.rs

```diff
@@ -87,6 +87,7 @@ impl Connection {
         self.backup_main(&destination)
     }
 
+    #[cfg(not(target_os = "linux"))]
```

Comment:
@mikayla-maki This seems like a (somewhat) good reason to bundle our own sqlite after all...

Reply:
This should only affect the syntax checking ‘sql!()’ macro that sqlez provides. Unfortunately, it’s required because the error offset function is too new for the version of sqlite that Ubuntu ships. Honestly, I should shorten the scope of this cfg to only cover the actual offending call.

Reply:
It would be better to define a minimum version of sqlite3 that is needed and switch to building the bundled version if the system version is lower than that (or just not build the function that depends on it if it's not important). If you don't already have an idea of how to do that the fish-shell devs created [rsconf](https://neosmart.net/blog/using-build-rs-to-integrate-rust-applications-with-system-libraries-like-a-pro/) which can fill that niche by providing cmake/meson-style system checks that can be used by build.rs. `has_symbol_in` or `has_symbols_in` would likely do the job if you know what symbol(s) the system sqlite3 library needs for this to work.